src/places.scss

```diff
@@ -85,4 +101,99 @@ $greyish: #a4a4a4;
       transform: scale(1)
     }
   }
+
+
+  // The following code is for the ap-input
+  // the input users will use hen generating their own
+  // searchbox
+  .ap-input {
+    position: relative;
+    width: 240px; // random value..
```

Comment:
There should be no custom widths set inside the default css class, we can just use width: 100%